### ISSUE TITLE
Browser support

### ISSUE THREAD

**Issue by antony** [isaacHagoel/svelte-dnd-action#253]
Hi - this looks fantastic. I'm currently using dragula across my apps and I'd definitely like to switch to this library instead.

One thing I don't see on the readme is anything about browser support. It would be good to know what browsers are expected to work, and which ones aren't.

**Comment by isaacHagoel** [comment]
good point. i believe it will probably work fine even on relatively old
browsers (based on the APIs the code is using) but I haven't tested that in
a while.
i can try to find time to play around with IE11 and IE9 and see what
happens.
modern browsers are supported for sure but that probably goes without
saying.
